pkg/controller/config/match/match.go

```diff
@@ -0,0 +1,65 @@
+package match
+
+import (
+	configv1alpha1 "github.com/open-policy-agent/gatekeeper/apis/config/v1alpha1"
+)
+
+type Operation string
+
+const (
```

Comment:
We should be using the same operation constants as `pkg/operations/operations.go`

Reply:
These two operations lists serve different purpose. https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/operations/operations.go#L16-L18 has `status` which won't make sense here to exclude. And `sync` won't make sense as an operation assigned to a pod by the deployment.

Reply:
+1

Reply:
`operation` aligns how we used in the past except with `status`. If we rename, how about `action` or `process`?

Reply:
`action` invokes `enforcementAction`, I don't think `process` is taken.

Reply:
Do we want to change it for config object too or keep operations here?
```
    - excludedNamespaces: ["kube-system", "star-excluded-namespace"]
      operations: ["*"]
```

to 

```
    - excludedNamespaces: ["kube-system", "star-excluded-namespace"]
      processes: ["*"]
```

Reply:
I like the idea of differentiating across the stack... if it's confusing to us, it's likely confusing to the user.

Reply:
+1 Let's separate them and use `process` for `config`. I will update the design doc accordingly.

Reply:
updated with `process` now in `config`